<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>638</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="txt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>521</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="new_2"/>
    <addaction name="separator"/>
    <addaction name="open"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="save_ano"/>
    <addaction name="separator"/>
    <addaction name="close"/>
    <addaction name="separator"/>
    <addaction name="export_2"/>
    <addaction name="separator"/>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="del"/>
    <addaction name="edit"/>
    <addaction name="add"/>
   </widget>
   <widget class="QMenu" name="menu_C">
    <property name="title">
     <string>审阅(C)</string>
    </property>
    <addaction name="find"/>
    <addaction name="separator"/>
    <addaction name="addup"/>
    <addaction name="separator"/>
    <addaction name="show"/>
    <addaction name="separator"/>
    <addaction name="protect"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>设置(O)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>工具栏</string>
     </property>
     <addaction name="action"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="about"/>
    <addaction name="separator"/>
    <addaction name="bug"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_C"/>
   <addaction name="menu_O"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="new_2"/>
   <addaction name="open"/>
   <addaction name="save"/>
   <addaction name="save_ano"/>
   <addaction name="close"/>
   <addaction name="export_2"/>
   <addaction name="separator"/>
   <addaction name="del"/>
   <addaction name="edit"/>
   <addaction name="add"/>
   <addaction name="separator"/>
   <addaction name="find"/>
   <addaction name="addup"/>
   <addaction name="show"/>
   <addaction name="protect"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="separator"/>
   <addaction name="about"/>
   <addaction name="bug"/>
   <addaction name="separator"/>
   <addaction name="quit"/>
  </widget>
  <action name="new_2">
   <property name="icon">
    <iconset>
     <normaloff>../icon/新建.png</normaloff>../icon/新建.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open">
   <property name="icon">
    <iconset>
     <normaloff>../icon/打开.ico</normaloff>../icon/打开.ico</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset>
     <normaloff>../icon/icon_保存.ico</normaloff>../icon/icon_保存.ico</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="save_ano">
   <property name="icon">
    <iconset>
     <normaloff>../icon/保存.ico</normaloff>../icon/保存.ico</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="close">
   <property name="icon">
    <iconset>
     <normaloff>../icon/关闭.ico</normaloff>../icon/关闭.ico</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="export_2">
   <property name="icon">
    <iconset>
     <normaloff>../icon/icon_使用文档.ico</normaloff>../icon/icon_使用文档.ico</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="quit">
   <property name="icon">
    <iconset>
     <normaloff>../icon/插头.ico</normaloff>../icon/插头.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="del">
   <property name="icon">
    <iconset>
     <normaloff>../icon/删除.ico</normaloff>../icon/删除.ico</iconset>
   </property>
   <property name="text">
    <string>删除记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="edit">
   <property name="icon">
    <iconset>
     <normaloff>../icon/suggestion.png</normaloff>../icon/suggestion.png</iconset>
   </property>
   <property name="text">
    <string>修改记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="add">
   <property name="icon">
    <iconset>
     <normaloff>../icon/方块+.ico</normaloff>../icon/方块+.ico</iconset>
   </property>
   <property name="text">
    <string>添加记录
</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="find">
   <property name="icon">
    <iconset>
     <normaloff>../icon/查询.ico</normaloff>../icon/查询.ico</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="addup">
   <property name="icon">
    <iconset>
     <normaloff>../icon/icon_添加.ico</normaloff>../icon/icon_添加.ico</iconset>
   </property>
   <property name="text">
    <string>统计</string>
   </property>
  </action>
  <action name="show">
   <property name="icon">
    <iconset>
     <normaloff>../icon/筛选.png</normaloff>../icon/筛选.png</iconset>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
  </action>
  <action name="protect">
   <property name="icon">
    <iconset>
     <normaloff>../icon/privacy_closed.png</normaloff>../icon/privacy_closed.png</iconset>
   </property>
   <property name="text">
    <string>保护文档</string>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset>
     <normaloff>../icon/更多.ico</normaloff>../icon/更多.ico</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="bug">
   <property name="icon">
    <iconset>
     <normaloff>../icon/技术服务.ico</normaloff>../icon/技术服务.ico</iconset>
   </property>
   <property name="text">
    <string>提供建议</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>../icon/min.png</normaloff>../icon/min.png</iconset>
   </property>
   <property name="text">
    <string>隐藏工具栏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
